cowbird/handlers/handler.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, settings, name, **kwargs):
         self.timeout = get_timeout(self.settings)
         for required_param in self.required_params:  # pylint: disable=E1101,no-member
             if required_param not in HANDLER_PARAMETERS:
-                raise Exception(f"Invalid handler parameter : {required_param}")
+                raise HandlerConfigurationException(f"Invalid handler parameter : {required_param}")
```

Comment:
Can you add a test for this use case? 
Codecov seems to indicate it was never evaluated.

Reply:
Fixed, there was actually already a case that was attempting to do this, but failed. It was triggering an `Exception` as expected, but not the right one, not reaching the targeted code here.